week_02/backjoon/ykshin/20200831_ykshin_oneLine.kt

```diff
@@ -0,0 +1,34 @@
+import java.io.BufferedReader
+import java.io.InputStreamReader
+import java.lang.StringBuilder
+import java.lang.System.`in`
+import java.util.*
+
+val sb :StringBuilder by lazy{
+    StringBuilder()
+}
+
+fun main() {
+    val br = BufferedReader(InputStreamReader(`in`))
+    val n = br.readLine().toInt()
+    val people = mutableListOf<Pair<Int,Int>>()
```

Comment:
nitpick,

```suggestion
    val people = mutableListOf<Pair<Int, Int>>()
```
구글에 검색해보았을 때는 한칸 띄는게 맞는 거 같은데? 맞니?? ㅋㅋㅋ

FYI, 회사에서 다른 선배개발자들에게 듣거나 멘토링 받거나, 피드백 받아서 알겠지만, 각 언어마다의 코드 컨벤션들이 있고, 요즘엔 IDE 들이 좋아져서, 틀리면 틀렸다고 바로바로 알려주고 명령어로 자동 포매팅도 해주다 보니, 코드 제출 전에 요런 것들 봐주면 더 좋을 것 같네~

Reply:
문제푸는것만 신경써서 해당 사항들은 무시했네용. 리포멧도 신경쓰겠습니당